src/app/workflow/versions/versions.component.html

```diff
@@ -81,6 +81,14 @@
         </label>
       </th> -->
 
+          <th>
+            <label
+              placement="top"
+              matTooltip="The metadata and descriptors for a workflow that has been snapshotted will not change over time."
```

Comment:
...for a workflow **version** that has been...

or

...for a version that has been...